handleSlimmeMeter.ino

```diff
@@ -10,12 +10,96 @@
 */  
 
 #if !defined(HAS_NO_SLIMMEMETER)
+//==================================================================================
+void handleSlimmemeter(){
```

Comment:
Ok, nieuwe standaard die ik gebruik is dat alle ‘{‘ en ‘}’ op een nieuwe regel en onder elkaar staan (behalve als het er tientallen zijn en het de leesbaarheid eerder tegen werkt dan beter maakt).

.. en zo niet dan toch wel een spatie vóór de ‘{‘ ..